### PR TITLE
Fallback image for blog card and blog page

### PR DESCRIPTION
**Pull request by rakeshPRaghu**
This PR will add a fallback image for the blog card and blog page.

1. Added a component which can display the fallback image if the given src fails
2. Removed duplicate images

### CODE REVIEW

File: website/src/components/BlogsSlider/style.ts [hunk 1]
```diff
@@ -10,7 +10,19 @@ const useStyles = makeStyles((theme: Theme) => ({
   media: {
     height: 0,
     paddingTop: "56.25%",
-    borderRadius: '24px 24px 24px 0px',
+    borderRadius: "24px 24px 24px 0px",
+    position: 'relative',
+    overflow: 'hidden',
+    '& img':{
+      minWidth: '100%',
```
Comment (IsAmrish) [inline website/src/components/BlogsSlider/style.ts:17]: here minWidth and maxWidth value is same, we should use `width: 100%` and remove other two `minWidth` and `maxWidth` property.

File: website/src/components/BlogsSlider/style.ts [hunk 2]
```diff
@@ -10,7 +10,19 @@ const useStyles = makeStyles((theme: Theme) => ({
...
+      minHeight: '100%',
+      maxWidth: '100%',
+      position: 'absolute',
+      left: '0',
+      top: '0',
+      objectFit: 'cover',
+      objectPosition: 'left'
```
Comment (IsAmrish) [inline website/src/components/BlogsSlider/style.ts:24]: why `left`, why not `middle`

File: website/src/components/MiniBlog/styles.ts [hunk 3]
```diff
@@ -98,8 +98,20 @@ const useStyles = makeStyles((theme: Theme) => ({
...
   media: {
     height: 0,
-    paddingTop: "56.25%", // 16:9
-    borderRadius: '24px 24px 24px 0px'
+    paddingTop: "56.25%",
+    borderRadius: "24px 24px 24px 0px",
+    position: 'relative',
+    overflow: 'hidden',
+    '& img':{
+      minWidth: '100%',
```
Comment (IsAmrish) [inline website/src/components/MiniBlog/styles.ts:106]: same comment as above

File: website/src/pages/Blog/styles.ts [hunk 4]
```diff
@@ -167,6 +167,18 @@ const useStyles = makeStyles((theme: Theme) => ({
     height: 0,
     paddingTop: "56.25%", // 16:9
     borderRadius: "24px 24px 24px 0px",
+    position: 'relative',
+    overflow: 'hidden',
+    '& img':{
+      minWidth: '100%',
```
Comment (IsAmrish) [inline website/src/pages/Blog/styles.ts:173]: here also

File: website/src/components/BlogsSlider/style.ts (hunk 2 above)
Comment (rakeshPRaghu) [reply website/src/components/BlogsSlider/style.ts:24]: It didn't look good when i tried with `center`

**Review by IsAmrish — APPROVED** [review]

**Review by AVRahul — APPROVED** [review]
lgtm

**Review by Pallavi-PH — APPROVED** [review]

**Review by satyapriyamishra222 — APPROVED** [review]

**Pull request merged**